pkg/controller/user-cluster-controller-manager/resources/resources/usersshkeys/networkpolicy.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package usersshkeys
+
+import (
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+)
+
+// NetworkPolicyCreator NetworkPolicy allows egress access to user ssh key agent.
```

Comment:
actually, it allows egress access **from** user ssh agent, right?

Reply:
Yes let me rephrase it